src/Equinox.Codec/Equinox.Codec.fsproj

```
@@ -15,6 +15,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
+    <PackageReference Include="MinVer" Version="1.0.0-beta.2+build.172" />
```

```suggestion
    <PackageReference Include="MinVer" Version="1.0.0-beta.2" />
```

Build metadata is not part of the version: https://www.nuget.org/packages/MinVer/1.0.0-beta.2

(Same comment applies to all the other package references.)